test_analysis/fake_data_stan_analysis_termite.py

```diff
@@ -0,0 +1,513 @@
+#
```

Comment:
is this the analysis script from termite? why is it in mermithid now?

Reply:
I'm not sure why this script was added, given that we also have a fake_data_stan_analysis.py script in mermithid for an example. I would suggest we remove fake_data_stan_analysis_termite.py. @casesyh, could you please comment?